src/process/conditions/index.ts

```diff
@@ -93,7 +93,10 @@ export const conditionalProcess = (context: ConditionsContext, isHidden: Conditi
   }
   let conditionalComp = scope.conditionals.find((cond) => cond.path === path);
   if (!conditionalComp) {
-    conditionalComp = { path, conditionallyHidden: false };
+    conditionalComp = {
```

Comment:
something seems fishy to me about this change. Why wouldn't one of the new pathing options work here for Panel? (i.e. `fullPath` or `fullDataPath` etc.)?